test/fixtures/for/in-auto-const/actual.js

```diff
@@ -1 +1 @@
-for k in {}: k
+for const k in {}: k
```

Comment:
Missing-const cases are already covered pretty extensively in the parser, did you want them replicated here?